miner/miner.go

```diff
@@ -425,8 +426,37 @@ func (m *Miner) mineOne(ctx context.Context, base *MiningBase) (*types.BlockMsg,
 		return nil, xerrors.Errorf("failed to get mining base info: %w", err)
 	}
 	if mbi == nil {
+		log.Warnf("mineOne: unexpectedly nil MiningBaseInfo for round %d, off tipset %d/%s", round, base.TipSet.Height(), base.TipSet.Key().String())
 		return nil, nil
 	}
+
+	// always write out a log from this point out
+	var winner *types.ElectionProof
+	lookBack := make(chan int64)
```

Comment:
I'd give this a buffer of one if we end up keeping it.

Reply:
We'll probably kill it, just double-checking on slack to be sure